src/sql/plot.py

```diff
@@ -455,7 +455,8 @@ def histogram(
             bottom += values_
 
         ax.set_title(f"Histogram from {table!r}")
-        ax.legend()
+        handles, labels = ax.get_legend_handles_labels()
+        ax.legend(handles[::-1], labels[::-1])
```

Comment:
What does this change do?

Reply:
Same as above. It reverses the order of the legend.